docs/docs/0.7.3/documentation/dps-and-data-model.md

```diff
@@ -66,7 +66,7 @@ networking:
 kuma-dp run \
   --name=redis-1 \
   --mesh=default \
-  --cp-address=http://127.0.0.1:5681 \
+  --cp-address=https://127.0.0.1:5678 \
```

Comment:
The port numbers are very difficult to follow. It will be a good idea to define a bunch of easy to follow ENV variables. Once defined use that throughout the document.
ex: 
CP_ADDRESS=https://127.0.0.1:5678
DP_ADDRESS=https://127.0.0.1:5681

Reply:
The document (dps-and-data-model.md) has:
2 occurrences of https://127.0.0.1:5678
0 occurrences of https://127.0.0.1:5681

I'm not sure how to define this in a way that will help a user and also not distract them from going back and forth to see the value. I'd love to see a PR with your suggestions!